run_migrations.py

```
@@ -0,0 +1,50 @@
+import logging
+from tortoise import Tortoise
+from app.config import TORTOISE_ORM
+
+
+AERICH_CONFIG = "aerich.ini"
+MIGRATIONS_DIR = "migrations"
+
+logger = logging.getLogger(__name__)
+
+
+async def ensure_tortoise_connected():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)